test/generator/files/track_files_test.rb

```diff
@@ -44,9 +43,9 @@ def test_tests_template
       class TestTrackFilesUseDefault
         def initialize
           @paths = FixturePaths
-          @slug = 'notemplate'
+          @exercise = Exercise.new(slug: 'notemplate')
```

Comment:
Probably. It would need to be `no-template`. This will need to happen later though as I'm trying not to mess with the pre-existing commits.